matrix/common/dss_zarr_store.py

```diff
@@ -49,14 +50,15 @@ class DSSZarrStore(MutableMapping):
     def _transform_key(self, key):
         return key.replace("/", self._separator_char)
 
-    def __init__(self, bundle_uuid, bundle_version=None, replica="aws"):
+    def __init__(self, request_hash, bundle_uuid, bundle_version=None, replica="aws"):
```

Comment:
Requiring `request_hash` seems to add in a new, unrelated concern to this class.

Reply:
I see your point and am OK with removing it. The goal was just to improve the logs by including the hash

Reply:
Gotcha. At some point we're going to have to give some code to people who want to read matrices from bundles directly. Ideally this class should just do it, along with the rest of the zarr package. So I'm reluctant to add in tighter coupling with matrix service-specific stuff.

Reply:
Removed